src/lib/services/project-service.ts

```diff
@@ -57,6 +59,20 @@ export interface AccessWithRoles {
     groups: IGroupModelWithProjectRole[];
 }
 
+export interface IStatusUpdate {
```

Comment:
I'm in two minds about this name. Aren't we doing project stats? Status sounds like on track/late etc.

Reply:
Sure, let's change the name.